NuGet.config

```diff
@@ -3,7 +3,8 @@
   <packageSources>
     <add key="ASP.NET 5 Dev" value="https://www.myget.org/F/aspnetvnext/" />
     <add key="ASP.NET 5" value="https://www.myget.org/F/aspnetrelease/" />
-    <add key="nuget.org feed" value="https://www.nuget.org/api/v2/" />
+    <add key="nuget.org feed" value="https://www.nuget.org/api/v2/" />    
+    <add key="UpdatedVersion" value="C:\Users\kartang\Documents\GitHub\NugetPackages" />
   </packageSources>
```

Comment:
We should remove this, but I am not sure if we need to point to the actual drop folder.